sal-service/src/main/java/org/ow2/proactive/sal/service/service/RepositoryService.java

```diff
@@ -773,6 +776,8 @@ public void cleanAll(String sessionId) throws NotConnectedException {
         if (!paGatewayService.isConnectionActive(sessionId)) {
             throw new NotConnectedException();
         }
+        LOGGER.info("Cleaning ProActive Vault...");
+        vaultService.removeAllSecrets(sessionId);
```

Comment:
No need to pass sessionId and check for connection as it is already done in the caller method.

Reply:
done